Vertex[] vertexes
Triangle[] triangles
byte[] scanImage
std_msgs/Header header
sensor_msgs/Image Image
Vertex scanOrigin
float32[] GLTransform